src/core/provider.cc

```diff
@@ -78,6 +78,14 @@ AddClassResults(
       }
 
       cls->set_value(static_cast<float>(probs[idx[k]]));
+
```

Comment:
Don't we need to conditionalize how we generate the class results based on the protocol?

Reply:
Will add condition on protocol version. Right now it prepares class results for both versions...